### ISSUE TITLE
Display Language doesn't update automatically

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#94201]
Re #93771

* Change Display Language in one instance, it requires restarting the app
* The second instance doesn't show any notification after sync update
* Restart the second instance uses the new display language

**Comment by sandy081** [comment]
Yeah there is no notification when display language is changed in the source file. Its not just by sync also if you open the file and change it.
